.github/workflows/build.yaml

```diff
@@ -37,7 +37,20 @@ jobs:
           version: edge
           static: ${{ matrix.os.static }}
       - run: npm install -g markdownlint-cli
+      - name: Restore rq cache
+        id: cache-rq
+        uses: actions/cache@v3
+        with:
+          path: ~/go/bin/rq
+          key: ${{ runner.os }}-rq
       - run: go install git.sr.ht/~charles/rq/cmd/rq@latest
+      - run: which rq
```

Comment:
Was this for debugging?

Reply:
Yeah, it's gone now. I have also updated to work for more than one macOS arch